src/helper.h

```diff
@@ -17,6 +17,13 @@ std::vector<char> binary_to_hex(char const *data, int len);
 
 std::vector<std::string> split(std::string const &input, std::string token);
 
```

Comment:
It's not supporting a specific class, but it's a free-standing function. Would you like to see the functions in helper.h put into a `namespace helper { }`?